script.js

```diff
@@ -1,91 +1,127 @@
 /**
  * Guess The Number Game
- * TODO: Get user value from input and save it to variable numberGuess
- * TODO: Generate a random number 1 to 100 and save it to variable correctNumber
- * TODO: Console whether the guess is too high, too low, or is correct inside playGame function
- * TODO: Create a function called displayResult to move the logic for if the guess is too high, too low, or correct
- * TODO: Complete the showYouWon, showNumberAbove, showNumberBelow
- * TODO: Use the showYouWon... functions within displayResult to display the correct dialog
- * TODO: Save the guess history in a variable called guess
- * TODO: Display the guess history using displayHistory() function
- * TODO: Use the initGame() function to restart the game
```

Comment:
Add the following list of functionality to the app:

- [ ] On enter or click the input value should disappear.

- [ ] When input is empty click or enter should not work.

- [ ] When you find the number you should not be able to click or enter. You can only give restart.

Reply:
First two tasks are done. Check the third task. Once you got the final result you are still able to input a number.

Reply:
- [ ] Your code is not resetting everything properly once you click restart. Please check that.